gcloud/bigtable/client.py

```diff
@@ -359,6 +363,10 @@ def stop(self):
         self._operations_stub_internal = None
         self._table_stub_internal = None
 
+    def __exit__(self, etype, e, traceback):
```

Comment:
I'm fairly certain Pylint will get angry about `e` as variable name. Instead you could just match the signature we typically use.

```python
def __exit__(self, exc_type, exc_val, exc_tb):
    ...
```